integration/fuse/src/main/java/alluxio/fuse/options/FuseOptions.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse.options;
+
+import alluxio.client.file.options.FileSystemOptions;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.conf.PropertyKey;
+import alluxio.exception.runtime.InvalidArgumentRuntimeException;
+import alluxio.fuse.AlluxioFuseUtils;
+import alluxio.jnifuse.utils.LibfuseVersion;
+
+import com.google.common.base.Preconditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Options for creating the Fuse filesystem.
+ */
+public class FuseOptions {
+  private static final Logger LOG = LoggerFactory.getLogger(FuseOptions.class);
+  private final FileSystemOptions mFileSystemOptions;
+  private final Set<String> mFuseMountOptions;
+  private final boolean mUpdateCheckEnabled;
+
+  /**
+   * Creates the FUSE options.
+   *
+   * @param conf alluxio configuration
+   * @return the file system options
+   */
+  public static FuseOptions create(AlluxioConfiguration conf) {
+    return create(conf, FileSystemOptions.create(conf), false);
+  }
+
+  /**
+   * Creates the FUSE options.
+   *
+   * @param conf alluxio configuration
+   * @param updateCheckEnabled whether to enable update check
+   * @return the file system options
+   */
+  public static FuseOptions create(AlluxioConfiguration conf, boolean updateCheckEnabled) {
+    return create(conf, FileSystemOptions.create(conf), updateCheckEnabled);
+  }
+
+  /**
+   * Creates the FUSE options.
+   *
+   * @param conf alluxio configuration
+   * @param fileSystemOptions the file system options
+   * @param updateCheckEnabled whether to enable update check
+   * @return the file system options
+   */
+  public static FuseOptions create(AlluxioConfiguration conf,
+      FileSystemOptions fileSystemOptions, boolean updateCheckEnabled) {
+    Set<String> mountOptions = conf.getList(PropertyKey.FUSE_MOUNT_OPTIONS)
+        .stream().filter(a -> !a.isEmpty()).collect(Collectors.toSet());
+    LibfuseVersion version = AlluxioFuseUtils.getLibfuseVersion(conf);
+    if (!conf.getBoolean(PropertyKey.FUSE_JNIFUSE_ENABLED)
```

Comment:
still support jnr fuse?

Reply:
Should remove JNR-FUSE but it's not removed yet